diff --git a/node_modules/@mswjs/interceptors/lib/node/chunk-EKNRB5ZS.mjs b/node_modules/@mswjs/interceptors/lib/node/chunk-EKNRB5ZS.mjs
index a1f1dd2..7d91fc5 100644
--- a/node_modules/@mswjs/interceptors/lib/node/chunk-EKNRB5ZS.mjs
+++ b/node_modules/@mswjs/interceptors/lib/node/chunk-EKNRB5ZS.mjs
@@ -471,13 +471,6 @@ var MockHttpSocket = class extends MockSocket {
     }
     const socket = this.createConnection();
     this.originalSocket = socket;
-    if ("_handle" in socket) {
-      Object.defineProperty(this, "_handle", {
-        value: socket._handle,
-        enumerable: true,
-        writable: true
-      });
-    }
     this.once("error", (error) => {
       socket.destroy(error);
     });
diff --git a/node_modules/@mswjs/interceptors/lib/node/chunk-N4ZZFE24.js b/node_modules/@mswjs/interceptors/lib/node/chunk-N4ZZFE24.js
index 6e68c6e..82d8fee 100644
--- a/node_modules/@mswjs/interceptors/lib/node/chunk-N4ZZFE24.js
+++ b/node_modules/@mswjs/interceptors/lib/node/chunk-N4ZZFE24.js
@@ -471,13 +471,6 @@ var MockHttpSocket = class extends MockSocket {
     }
     const socket = this.createConnection();
     this.originalSocket = socket;
-    if ("_handle" in socket) {
-      Object.defineProperty(this, "_handle", {
-        value: socket._handle,
-        enumerable: true,
-        writable: true
-      });
-    }
     this.once("error", (error) => {
       socket.destroy(error);
     });
diff --git a/node_modules/@mswjs/interceptors/src/interceptors/ClientRequest/MockHttpSocket.ts b/node_modules/@mswjs/interceptors/src/interceptors/ClientRequest/MockHttpSocket.ts
index c4fe832..ae9776c 100644
--- a/node_modules/@mswjs/interceptors/src/interceptors/ClientRequest/MockHttpSocket.ts
+++ b/node_modules/@mswjs/interceptors/src/interceptors/ClientRequest/MockHttpSocket.ts
@@ -187,20 +187,6 @@ export class MockHttpSocket extends MockSocket {
     const socket = this.createConnection()
     this.originalSocket = socket
 
-    /**
-     * @note Inherit the original socket's connection handle.
-     * Without this, each push to the mock socket results in a
-     * new "connection" listener being added (i.e. buffering pushes).
-     * @see https://github.com/nodejs/node/blob/b18153598b25485ce4f54d0c5cb830a9457691ee/lib/net.js#L734
-     */
-    if ('_handle' in socket) {
-      Object.defineProperty(this, '_handle', {
-        value: socket._handle,
-        enumerable: true,
-        writable: true,
-      })
-    }
-
     // If the developer destroys the socket, destroy the original connection.
     this.once('error', (error) => {
       socket.destroy(error)
