diff --git a/node_modules/@originjs/vite-plugin-federation/dist/index.mjs b/node_modules/@originjs/vite-plugin-federation/dist/index.mjs
index 6118783..41d57d6 100644
--- a/node_modules/@originjs/vite-plugin-federation/dist/index.mjs
+++ b/node_modules/@originjs/vite-plugin-federation/dist/index.mjs
@@ -663,7 +663,10 @@ const {${imports}} = ${defaultImportDeclaration};
           );
         }
         if (requiresRuntime || hasImportShared || modify) {
-          return magicString.toString();
+          return {
+            code: magicString.toString(),
+            map: magicString.generateMap({ hires: true })
+          };
         }
       }
     }
