diff --git a/node_modules/@segment/top-domain/lib/index.js b/node_modules/@segment/top-domain/lib/index.js
index ae67f2e..1913b53 100644
--- a/node_modules/@segment/top-domain/lib/index.js
+++ b/node_modules/@segment/top-domain/lib/index.js
@@ -4,7 +4,6 @@
  * Module dependencies.
  */
 
-var parse = require('component-url').parse;
 var cookie = require('component-cookie');
 
 /**
@@ -55,6 +54,12 @@ function domain(url) {
   return '';
 }
 
+function legacyGetHostname(href) {
+  const l = document.createElement("a");
+  l.href = href;
+  return l.hostname;
+};
+
 /**
  * Levels returns all levels of the given url.
  *
@@ -63,7 +68,7 @@ function domain(url) {
  * @api public
  */
 domain.levels = function(url) {
-  var host = parse(url).hostname;
+  var host = typeof window.URL !== 'function' ? legacyGetHostname(url) : new URL(url).hostname;
   var parts = host.split('.');
   var last = parts[parts.length - 1];
   var levels = [];
