diff --git a/node_modules/conventional-github-releaser/src/cli.js b/node_modules/conventional-github-releaser/src/cli.js
index ea15f54..8c48a98 100755
--- a/node_modules/conventional-github-releaser/src/cli.js
+++ b/node_modules/conventional-github-releaser/src/cli.js
@@ -38,6 +38,8 @@ const cli = meow({
 
       -d, --draft               Publishes a draft instead of a real release
                                 Default: false
+
+      -m, --package             Package name for monorepos
   `,
   flags: {
     url: {
@@ -80,7 +82,11 @@ const cli = meow({
       alias: 'd',
       default: false,
       type: 'boolean'
-    }
+    },
+    package: {
+      alias: 'm',
+      type: 'string'
+    },
   }
 })
 
@@ -123,7 +129,8 @@ const changelogOpts = {
     path: flags.pkg
   },
   releaseCount: flags.releaseCount,
-  draft: flags.draft
+  draft: flags.draft,
+  lernaPackage: flags.package
 }
 
 if (flags.verbose) {
diff --git a/node_modules/conventional-github-releaser/src/index.js b/node_modules/conventional-github-releaser/src/index.js
index c62bad7..b95713d 100644
--- a/node_modules/conventional-github-releaser/src/index.js
+++ b/node_modules/conventional-github-releaser/src/index.js
@@ -47,7 +47,7 @@ function conventionalGithubReleaser (auth, changelogOpts, context, gitRawCommits
   // ignore the default header partial
   writerOpts.headerPartial = writerOpts.headerPartial || ''
 
-  Q.nfcall(gitSemverTags)
+  Q.nfcall(gitSemverTags, { lernaTags: !!changelogOpts.lernaPackage, package: changelogOpts.lernaPackage})
     .then(function (tags) {
       if (!tags || !tags.length) {
         setImmediate(userCb, new Error('No semver tags found'))
@@ -75,7 +75,8 @@ function conventionalGithubReleaser (auth, changelogOpts, context, gitRawCommits
 
           const url = `repos/${context.owner}/${context.repository}/releases`
           const options = {
-            endpoint: auth.url,
+            baseUrl: auth.url,
+            method: 'POST',
             body: {
               body: chunk.log,
               draft: changelogOpts.draft || false,
