diff --git a/node_modules/nx/src/command-line/yargs-utils/shared-options.js b/node_modules/nx/src/command-line/yargs-utils/shared-options.js
index 0f92f37..5b625b1 100644
--- a/node_modules/nx/src/command-line/yargs-utils/shared-options.js
+++ b/node_modules/nx/src/command-line/yargs-utils/shared-options.js
@@ -150,6 +150,10 @@ function withRunManyOptions(yargs) {
         describe: '[deprecated] Run the target on all projects in the workspace',
         type: 'boolean',
         default: true,
+    })
+        .option('silent', {
+        type: 'boolean',
+        default: false,
     });
 }
 exports.withRunManyOptions = withRunManyOptions;
diff --git a/node_modules/nx/src/tasks-runner/life-cycles/static-run-many-terminal-output-life-cycle.js b/node_modules/nx/src/tasks-runner/life-cycles/static-run-many-terminal-output-life-cycle.js
index a9e8197..c6a9430 100644
--- a/node_modules/nx/src/tasks-runner/life-cycles/static-run-many-terminal-output-life-cycle.js
+++ b/node_modules/nx/src/tasks-runner/life-cycles/static-run-many-terminal-output-life-cycle.js
@@ -23,6 +23,10 @@ class StaticRunManyTerminalOutputLifeCycle {
         this.allCompletedTasks = new Map();
     }
     startCommand() {
+        if (this.args.silent) {
+            return;
+        }
+
         if (this.projectNames.length <= 0) {
             output_1.output.logSingleLine(`No projects with ${(0, formatting_utils_1.formatTargetsAndProjects)(this.projectNames, this.args.targets, this.tasks)} were run`);
             return;
@@ -44,6 +48,10 @@ class StaticRunManyTerminalOutputLifeCycle {
         output_1.output.addVerticalSeparatorWithoutNewLines('cyan');
     }
     endCommand() {
+        if (this.args.silent) {
+            return;
+        }
+        
         output_1.output.addNewline();
         if (this.failedTasks.length === 0) {
             output_1.output.addVerticalSeparatorWithoutNewLines('green');
@@ -92,9 +100,11 @@ class StaticRunManyTerminalOutputLifeCycle {
         }
     }
     printTaskTerminalOutput(task, cacheStatus, terminalOutput) {
-        const args = (0, utils_1.getPrintableCommandArgsForTask)(task);
-        output_1.output.logCommand(args.join(' '), cacheStatus);
-        output_1.output.addNewline();
+        if (this.args.silent !== true) {
+            const args = (0, utils_1.getPrintableCommandArgsForTask)(task);
+            output_1.output.logCommand(args.join(' '), cacheStatus);
+            output_1.output.addNewline();
+        }
         process.stdout.write(terminalOutput);
     }
 }
