diff --git a/node_modules/storejs/dist/store.js b/node_modules/storejs/dist/store.js
index 76ebbe4..605d622 100644
--- a/node_modules/storejs/dist/store.js
+++ b/node_modules/storejs/dist/store.js
@@ -91,7 +91,8 @@
       return this;
     },
     get: function get(key) {
-      if (!key) {
+      // if no 'key' is passed return all
+      if (key === undefined) {
         var ret = {};
         this.forEach(function (key, val) {
           return ret[key] = val;
